pyseer/input.py

```diff
@@ -359,7 +359,7 @@ def read_variant(infile, p, var_type, burden, burden_regions,
                     for variant in infile.fetch(region.group(1),
                                                 int(region.group(2)) - 1,
                                                 int(region.group(3))):
-                        var_sub_name = read_vcf_var(variant, d)
+                        var_sub_name = read_vcf_var(variant, d, keep_list)
```

Comment:
I haven't tried if this change actually works yet, but in theory we should pass the keep list here as well, right?

Reply:
Yes, this makes sense!